.rubocop.yml

```diff
@@ -82,6 +86,12 @@ Metrics/ParameterLists:
   Max: 4
 Metrics/PerceivedComplexity:
   Max: 8
+Naming/FileName:
+  Enabled: false
+Naming/HeredocDelimiterCase:
+  Enabled: false
+Naming/HeredocDelimiterNaming:
+  Enabled: false
```

Comment:
What fails for this?

Reply:
- 'features/support/helpers.rb'
 - 'test/test_redcarpet.rb'
 - 'test/test_tags.rb'

are flagged for presence of `EOS` as the delimiter